### PR TITLE
fix(psh): Provide default psh cache expiry for /__docs

### PR DESCRIPTION
**Pull request by VancheeZze**
## Changes
- `expires` key was added for `/__docs` path.

#### Test if `expires` prop works:
- Deploy a site to p.sh using updated `@bodiless/psh` package.
- Open Docs
- Check the `network` tab in the chrome console.
- You should see the `Cache-Control` and `Expires` headers enabled.

## Related Issues
Fixes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/172

### CODE REVIEW

File: packages/bodiless-psh/src/bodiless-psh-init.ts [hunk 1]
```diff
@@ -56,7 +56,6 @@ const mergeByKey = (Source: any, Destination: any, Whitelist: any) => {
             console.log(`Key '${key}' is not whitelisted and not found in '@bodiless/psh' defaults. The value of ${Destination[key]} will be used.`);
             Object.assign(result, { [key]: Destination[key] });
           } else {
-            console.log(`Key '${key}' is not whitelisted. The default value of '${Source[key]}' from '@bodiless/psh' will be used.`);
```
Comment (hvanyo) [inline packages/bodiless-psh/src/bodiless-psh-init.ts:59]: @VancheeZze 
Will this produce a warning elsewhere if the key is "not whitelist" and default value != site value?

Comment (VancheeZze) [reply packages/bodiless-psh/src/bodiless-psh-init.ts:59]: Yep, Just added it.

File: packages/bodiless-psh/src/bodiless-psh-init.ts [hunk 2]
```diff
@@ -56,7 +56,9 @@ const mergeByKey = (Source: any, Destination: any, Whitelist: any) => {
...
+            if (Source[key] !== Destination[key]) {
+              console.log(`Key '${key}' is not whitelisted and default value of '${Source[key]}' is different from site level value '${Destination[key]}'. Default value will be used. `);
```
Comment (dewen) [inline packages/bodiless-psh/src/bodiless-psh-init.ts:60]: Seems this doesn't work for array checking?

Comment (VancheeZze) [reply packages/bodiless-psh/src/bodiless-psh-init.ts:60]: It only happens to the array of simple items ( not pair/key arrays - see Supported YAML types here https://github.com/nodeca/js-yaml#supported-yaml-types ). I think this is expected.

**Pull request merged**